math/binomialcoefficient_test.go

```diff
@@ -0,0 +1,42 @@
+// binomialcoefficient_test.go
+// description: Returns C(n, k)
+// author: Akshay Dubey (https://github.com/itsAkshayDubey)
+// see binomialcoefficient.go
+
+package math_test
+
+import (
+	"testing"
+
+	"github.com/TheAlgorithms/Go/math"
+)
+
+func TestCombinations(t *testing.T) {
+	var tests = []struct {
+		name          string
+		n             int
+		k             int
+		expectedValue int
+		expectedError error
+	}{
+		{"n = 5, k = 2", 5, 2, 10, nil},
+		{"n = 7, k = 4", 7, 4, 35, nil},
+		{"n = 0, k = 0", 0, 0, 1, nil},
+		{"n = -1, k = 1", -1, 1, -1, math.ErrPosArgsOnly},
+		{"n = 1, k = -1", 1, -1, -1, math.ErrPosArgsOnly},
+		{"n = -1, k = -1", -1, -1, -1, math.ErrPosArgsOnly},
+	}
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			result, error := math.Combinations(test.n, test.k)
```

Comment:
This is good but `error` is a name for an actual interface which is being shadowed here.

```suggestion
			result, err := math.Combinations(test.n, test.k)
```